build-system/tasks/process-github-issues.js

```diff
@@ -262,13 +276,15 @@ function githubRequest(path, opt_method, opt_data, typeRequest) {
     if (typeRequest === 'milestone') {
       options.body['milestone'] = opt_data;
     }
-    else options.body = opt_data;
+    else {
+      options.body = opt_data;
+    }
   }
   return request(options)
 }
 
 gulp.task('process-github-issues', 'Get issues data', processIssues, {
```

Comment:
Change this to something more descriptive than "Get issues data", since this is what shows up when someone runs `gulp help`.

Perhaps "Automatically updates the milestones of all open issues at github.com/ampproject/amphtml."?